app/controllers/events_controller.rb

```diff
@@ -99,4 +102,12 @@ def authorize_user_manage
       redirect_to(root_path)
     end
   end
+
+  def get_role
+    @roles = {}
+    EventStaff.roles.each do |key, value|
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.